common/changes/@itwin/core-react/disable-button-propagation_2024-04-29-12-44.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-react",
+      "comment": "Disabled `onMouseDown` event propagation for `ExpansionToggle`.",
```

Comment:
Would you mind leaving a note in `NextVersion.md` as well (together w/ #814).

Reply:
Updated `NextVersion.md`.